shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/domain/usecase/GetCurrentUserUseCase.kt

```diff
@@ -0,0 +1,19 @@
+package co.nimblehq.mark.kmmic.domain.usecase
+
+import co.nimblehq.mark.kmmic.domain.model.User
+import co.nimblehq.mark.kmmic.domain.repository.UserRepository
+import kotlinx.coroutines.flow.Flow
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.inject
+
+interface GetCurrentUserUseCase {
+    fun call(): Flow<User>
+}
+
+class GetCurrentUserUseCaseImpl : GetCurrentUserUseCase, KoinComponent {
+    private val userRepository: UserRepository by inject()
+
+    override fun call(): Flow<User> {
```

Comment:
Please use `revoke`

Reply:
@suho updated at [3225037](https://github.com/markgravity/kmm-ic/pull/72/commits/3225037d11388c90ad505fdf1de85c0ac1d7d6f2)

Reply:
@markgravity Please pay attention when updating the code, `invoke` should be use with operator in kotlin.